src/components/Dashboard/FilingHistoryList/filings/AlterationFiling.vue

```diff
@@ -0,0 +1,153 @@
+<template>
```

Comment:
This is the alteration filing component.

As you see below, not all slots are provided -- so the fallback slot content from FilingTemplate will be used.